setup.cfg

```
@@ -1,7 +1,7 @@
 [metadata]
 name = napari-process-points-and-surfaces
-version = 0.1.2
-author = Robert Haase
+version = 0.1.3
```

```suggestion
version = 0.1.2
```
I'd like to bump the version when a new release is deployed